R/t-distribution

```diff
@@ -0,0 +1,270 @@
+# App: t_distribution
+# Last update: 15 May 22
+
+library(shiny)
+library(miniUI)
+library(tidyverse)
+library(glue)
+library(Cairo)
+
+options(shiny.usecairo = TRUE)
+options(shiny.autoreload = TRUE)
+
+PRECISION <- 0.05
+EXTRA_POINTS_AROUND_CRITICAL_VALUES = c(-0.001, 0, 0.001)
+STARTING_T_VALUE <- 2.25
+STARTING_DF <- 2
+STARTING_P_VALUE <-
+  pt(STARTING_T_VALUE / 2, STARTING_DF, lower.tail = STARTING_T_VALUE <= 0) %>% round(3)
+RANGE <- c(-4, 4) # range for T value
```

Comment:
We'll need to add dynamic range, as with low df the critical values fall outside of the -4, 4 range.

`min/max(range, actual)` in the render should do the trick